Jenkinsfile

```diff
@@ -482,11 +482,11 @@ noToolsScalaVersions.each { scalaVersion ->
   quickMatrix.add([task: "test-suite-ecma-script6", scala: scalaVersion, java: mainJavaVersion, testSuite: "testSuite"])
 }
 allJavaVersions.each { javaVersion ->
-  quickMatrix.add([task: "tools-cli-stubs-sbtplugin", scala: "2.10.7", java: javaVersion])
+  quickMatrix.add([task: "tools-cli-stubs-sbtplugin", scala: "2.10.7", sbt_version_override: "0.13.17", java: javaVersion])
 }
 quickMatrix.add([task: "tools-cli-stubs", scala: "2.12.8", java: mainJavaVersion])
 quickMatrix.add([task: "partestc", scala: "2.11.0", java: mainJavaVersion])
-quickMatrix.add([task: "sbtplugin-test", toolsscala: "2.10.7", sbt_version_override: "", java: mainJavaVersion])
+quickMatrix.add([task: "sbtplugin-test", toolsscala: "2.10.7", sbt_version_override: "0.13.17", java: mainJavaVersion])
 quickMatrix.add([task: "sbtplugin-test", toolsscala: "2.12.8", sbt_version_override: "1.0.0", java: mainJavaVersion])
```

Comment:
Could you try emptying the `sbt_version_override` for the 2.12.8 case, and change https://github.com/scala-js/scala-js/blob/0.6.x/sbt-plugin-test/project/build.properties to use 1.0.0?

Reply:
Fixed